[2024-06-13T19:33:07.965+0000] {processor.py:161} INFO - Started process (PID=167) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:07.984+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:33:08.019+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:33:08.017+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:08.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:08.158+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.252 seconds
[2024-06-13T19:33:38.966+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:38.978+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:33:38.999+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:33:38.997+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:39.044+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:33:39.130+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.210 seconds
[2024-06-13T19:34:10.032+0000] {processor.py:161} INFO - Started process (PID=190) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:10.035+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:34:10.039+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:34:10.038+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:10.047+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:10.071+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.048 seconds
[2024-06-13T19:34:41.003+0000] {processor.py:161} INFO - Started process (PID=201) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:41.024+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:34:41.032+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:34:41.030+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:41.054+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:34:41.111+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.127 seconds
[2024-06-13T19:35:11.937+0000] {processor.py:161} INFO - Started process (PID=212) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:11.938+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:35:11.942+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:35:11.941+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:11.952+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:11.976+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.054 seconds
[2024-06-13T19:35:43.018+0000] {processor.py:161} INFO - Started process (PID=223) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:43.022+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:35:43.026+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:35:43.025+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:43.033+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:35:43.071+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.068 seconds
[2024-06-13T19:36:13.443+0000] {processor.py:161} INFO - Started process (PID=234) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:13.445+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:36:13.448+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:36:13.448+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:13.455+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:13.515+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.082 seconds
[2024-06-13T19:36:39.104+0000] {processor.py:161} INFO - Started process (PID=245) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:39.110+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:36:39.120+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:36:39.119+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:39.142+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:36:39.138+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 21
    task_id = 'spotify_etl'
              ^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-06-13T19:36:39.147+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:36:39.400+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.340 seconds
[2024-06-13T19:37:10.218+0000] {processor.py:161} INFO - Started process (PID=256) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:10.221+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:37:10.225+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:10.224+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:10.236+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:10.234+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 21
    task_id = 'spotify_etl'
              ^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
[2024-06-13T19:37:10.237+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:10.269+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.061 seconds
[2024-06-13T19:37:14.289+0000] {processor.py:161} INFO - Started process (PID=261) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:14.291+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:37:14.293+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:14.293+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:14.308+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:14.305+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:37:14.326+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:14.370+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.091 seconds
[2024-06-13T19:37:44.719+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:44.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:37:44.724+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:44.723+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:44.734+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:37:44.732+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:37:44.735+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:37:44.763+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.053 seconds
[2024-06-13T19:38:15.034+0000] {processor.py:161} INFO - Started process (PID=283) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:15.035+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:38:15.038+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:38:15.037+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:15.046+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:38:15.044+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:38:15.047+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:15.077+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.053 seconds
[2024-06-13T19:38:45.361+0000] {processor.py:161} INFO - Started process (PID=294) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:45.363+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:38:45.366+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:38:45.366+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:45.378+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:38:45.376+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:38:45.380+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:38:45.412+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.062 seconds
[2024-06-13T19:39:15.589+0000] {processor.py:161} INFO - Started process (PID=305) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:15.591+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:39:15.594+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:39:15.594+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:15.606+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:39:15.604+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:39:15.608+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:15.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.057 seconds
[2024-06-13T19:39:45.852+0000] {processor.py:161} INFO - Started process (PID=316) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:45.854+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:39:45.857+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:39:45.856+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:45.867+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:39:45.865+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:39:45.869+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:39:45.896+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.056 seconds
[2024-06-13T19:40:16.175+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:16.177+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:40:16.179+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:40:16.179+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:16.187+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:40:16.185+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:40:16.188+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:16.217+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.051 seconds
[2024-06-13T19:40:46.443+0000] {processor.py:161} INFO - Started process (PID=338) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:46.446+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:40:46.449+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:40:46.449+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:46.464+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:40:46.461+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:40:46.470+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:40:46.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.078 seconds
[2024-06-13T19:41:16.817+0000] {processor.py:161} INFO - Started process (PID=350) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:16.819+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:41:16.822+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:41:16.821+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:16.831+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:41:16.829+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:41:16.832+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:16.861+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.051 seconds
[2024-06-13T19:41:47.112+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:47.113+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:41:47.116+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:41:47.116+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:47.125+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:41:47.123+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:41:47.126+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:41:47.155+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.053 seconds
[2024-06-13T19:42:17.408+0000] {processor.py:161} INFO - Started process (PID=372) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:17.410+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:42:17.413+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:42:17.413+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:17.427+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:42:17.424+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:42:17.428+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:17.519+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.124 seconds
[2024-06-13T19:42:47.670+0000] {processor.py:161} INFO - Started process (PID=383) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:47.672+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:42:47.675+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:42:47.675+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:47.686+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:42:47.684+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:42:47.688+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:42:47.717+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.057 seconds
[2024-06-13T19:43:17.983+0000] {processor.py:161} INFO - Started process (PID=394) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:17.985+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:43:17.988+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:43:17.988+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:17.999+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:43:17.997+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:43:18.000+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:18.030+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.057 seconds
[2024-06-13T19:43:48.281+0000] {processor.py:161} INFO - Started process (PID=405) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:48.283+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:43:48.287+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:43:48.286+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:48.300+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:43:48.297+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:43:48.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:43:48.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.075 seconds
[2024-06-13T19:44:18.596+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:18.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:44:18.602+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:44:18.601+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:18.615+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:44:18.612+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:44:18.616+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:18.656+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.075 seconds
[2024-06-13T19:44:49.007+0000] {processor.py:161} INFO - Started process (PID=427) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:49.009+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:44:49.011+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:44:49.011+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:49.020+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:44:49.018+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:44:49.021+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:44:49.049+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.051 seconds
[2024-06-13T19:45:19.270+0000] {processor.py:161} INFO - Started process (PID=439) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:19.272+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:45:19.276+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:45:19.275+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:19.285+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:45:19.283+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:45:19.286+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:19.315+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.053 seconds
[2024-06-13T19:45:49.603+0000] {processor.py:161} INFO - Started process (PID=451) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:49.605+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:45:49.608+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:45:49.608+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:49.618+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:45:49.616+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:45:49.619+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:45:49.648+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.052 seconds
[2024-06-13T19:46:19.914+0000] {processor.py:161} INFO - Started process (PID=462) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:19.915+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:46:19.918+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:46:19.918+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:19.929+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:46:19.926+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:46:19.930+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:19.964+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.059 seconds
[2024-06-13T19:46:50.179+0000] {processor.py:161} INFO - Started process (PID=473) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:50.181+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:46:50.185+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:46:50.184+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:50.197+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:46:50.194+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:46:50.198+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:46:50.232+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.063 seconds
[2024-06-13T19:47:20.499+0000] {processor.py:161} INFO - Started process (PID=484) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:20.500+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:47:20.503+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:47:20.503+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:20.512+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:47:20.510+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:47:20.513+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:20.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.054 seconds
[2024-06-13T19:47:50.797+0000] {processor.py:161} INFO - Started process (PID=495) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:50.799+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:47:50.801+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:47:50.801+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:50.812+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:47:50.809+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:47:50.813+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:47:50.840+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.052 seconds
[2024-06-13T19:48:21.149+0000] {processor.py:161} INFO - Started process (PID=506) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:21.151+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:48:21.154+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:48:21.154+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:21.165+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:48:21.162+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:48:21.166+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:21.198+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.058 seconds
[2024-06-13T19:48:51.437+0000] {processor.py:161} INFO - Started process (PID=517) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:51.439+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:48:51.442+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:48:51.441+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:51.452+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:48:51.450+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:48:51.454+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:48:51.484+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.057 seconds
[2024-06-13T19:49:21.728+0000] {processor.py:161} INFO - Started process (PID=528) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:21.730+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:49:21.733+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:49:21.733+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:21.744+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:49:21.741+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:49:21.745+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:21.775+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.060 seconds
[2024-06-13T19:49:52.012+0000] {processor.py:161} INFO - Started process (PID=539) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:52.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:49:52.017+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:49:52.017+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:52.030+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:49:52.027+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:49:52.032+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:49:52.065+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.063 seconds
[2024-06-13T19:50:22.344+0000] {processor.py:161} INFO - Started process (PID=550) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:22.346+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:50:22.349+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:50:22.349+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:22.360+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:50:22.357+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:50:22.361+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:22.389+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.054 seconds
[2024-06-13T19:50:52.640+0000] {processor.py:161} INFO - Started process (PID=561) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:52.642+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:50:52.645+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:50:52.645+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:52.656+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:50:52.653+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:50:52.657+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:50:52.686+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.058 seconds
[2024-06-13T19:51:22.977+0000] {processor.py:161} INFO - Started process (PID=572) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:22.979+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:51:22.982+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:51:22.981+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:22.993+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:51:22.991+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:51:22.995+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:23.023+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.056 seconds
[2024-06-13T19:51:53.252+0000] {processor.py:161} INFO - Started process (PID=583) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:53.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:51:53.257+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:51:53.256+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:53.268+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:51:53.266+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:51:53.269+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:51:53.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.058 seconds
[2024-06-13T19:52:23.544+0000] {processor.py:161} INFO - Started process (PID=594) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:23.546+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:52:23.549+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:52:23.548+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:23.561+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:52:23.559+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:52:23.562+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:23.589+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.055 seconds
[2024-06-13T19:52:53.887+0000] {processor.py:161} INFO - Started process (PID=606) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:53.889+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:52:53.892+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:52:53.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:53.905+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:52:53.902+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:52:53.907+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:52:53.938+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.062 seconds
[2024-06-13T19:53:24.205+0000] {processor.py:161} INFO - Started process (PID=616) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:24.207+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:53:24.210+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:53:24.209+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:24.221+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:53:24.219+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:53:24.223+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:24.253+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.058 seconds
[2024-06-13T19:53:54.478+0000] {processor.py:161} INFO - Started process (PID=627) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:54.480+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:53:54.482+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:53:54.482+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:54.493+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:53:54.491+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:53:54.494+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:53:54.524+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.056 seconds
[2024-06-13T19:54:24.803+0000] {processor.py:161} INFO - Started process (PID=639) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:24.804+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:54:24.807+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:54:24.807+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:24.818+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:54:24.816+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:54:24.820+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:24.848+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.056 seconds
[2024-06-13T19:54:55.143+0000] {processor.py:161} INFO - Started process (PID=650) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:55.145+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:54:55.148+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:54:55.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:55.159+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:54:55.156+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:54:55.161+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:54:55.192+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.060 seconds
[2024-06-13T19:56:29.472+0000] {processor.py:161} INFO - Started process (PID=662) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:56:29.475+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:56:29.482+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:56:29.481+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:56:29.522+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:56:29.517+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:56:29.525+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:56:29.586+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.167 seconds
[2024-06-13T19:57:07.415+0000] {processor.py:161} INFO - Started process (PID=673) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:07.425+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:57:07.442+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:57:07.442+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:07.608+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:57:07.604+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:57:07.611+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:07.694+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.345 seconds
[2024-06-13T19:57:42.578+0000] {processor.py:161} INFO - Started process (PID=684) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:42.586+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T19:57:42.603+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:57:42.602+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:42.615+0000] {logging_mixin.py:188} INFO - [2024-06-13T19:57:42.612+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T19:57:42.624+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T19:57:42.769+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.450 seconds
[2024-06-13T20:02:56.051+0000] {processor.py:161} INFO - Started process (PID=696) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:02:56.052+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:02:56.055+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:02:56.055+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:02:56.069+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:02:56.066+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:02:56.075+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:02:56.124+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.099 seconds
[2024-06-13T20:03:26.952+0000] {processor.py:161} INFO - Started process (PID=713) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:03:26.956+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:03:26.964+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:03:26.963+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:03:26.996+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:03:26.990+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:03:27.001+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:03:27.083+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.157 seconds
[2024-06-13T20:27:55.972+0000] {processor.py:161} INFO - Started process (PID=724) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:27:56.003+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:27:56.029+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:27:56.024+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:27:56.142+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:27:56.139+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:27:56.144+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:27:56.197+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.272 seconds
[2024-06-13T20:28:26.445+0000] {processor.py:161} INFO - Started process (PID=729) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:28:26.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:28:26.459+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:28:26.457+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:28:26.489+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:28:26.482+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:28:26.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:28:26.640+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.226 seconds
[2024-06-13T20:43:44.624+0000] {processor.py:161} INFO - Started process (PID=742) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:43:44.625+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:43:44.628+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:43:44.628+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:43:44.636+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:43:44.634+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:43:44.639+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:43:45.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.661 seconds
[2024-06-13T20:44:16.218+0000] {processor.py:161} INFO - Started process (PID=759) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:44:16.224+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T20:44:16.247+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:44:16.246+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:44:16.292+0000] {logging_mixin.py:188} INFO - [2024-06-13T20:44:16.285+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T20:44:16.298+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T20:44:16.385+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.257 seconds
[2024-06-13T21:00:33.659+0000] {processor.py:161} INFO - Started process (PID=765) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:00:33.704+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:00:33.715+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:00:33.714+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:00:33.770+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:00:33.766+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:00:33.772+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:00:33.874+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.267 seconds
[2024-06-13T21:01:04.888+0000] {processor.py:161} INFO - Started process (PID=783) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:04.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:01:04.912+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:01:04.908+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:04.969+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:01:04.956+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:01:04.980+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:05.129+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.347 seconds
[2024-06-13T21:01:35.608+0000] {processor.py:161} INFO - Started process (PID=794) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:35.614+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:01:35.625+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:01:35.623+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:35.667+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:01:35.658+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:01:35.672+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:01:35.782+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.211 seconds
[2024-06-13T21:02:06.303+0000] {processor.py:161} INFO - Started process (PID=805) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:06.309+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:02:06.319+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:02:06.317+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:06.356+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:02:06.348+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:02:06.360+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:06.457+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.186 seconds
[2024-06-13T21:02:36.771+0000] {processor.py:161} INFO - Started process (PID=816) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:36.776+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:02:36.784+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:02:36.782+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:36.816+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:02:36.809+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:02:36.820+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:02:36.913+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.175 seconds
[2024-06-13T21:03:07.358+0000] {processor.py:161} INFO - Started process (PID=827) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:07.363+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:03:07.370+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:03:07.369+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:07.401+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:03:07.394+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:03:07.405+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:07.482+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.148 seconds
[2024-06-13T21:03:37.784+0000] {processor.py:161} INFO - Started process (PID=838) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:37.790+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:03:37.801+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:03:37.800+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:37.850+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:03:37.841+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:03:37.856+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:03:37.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.225 seconds
[2024-06-13T21:04:08.324+0000] {processor.py:161} INFO - Started process (PID=849) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:08.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:04:08.335+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:04:08.334+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:08.362+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:04:08.357+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:04:08.366+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:08.436+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.137 seconds
[2024-06-13T21:04:38.796+0000] {processor.py:161} INFO - Started process (PID=859) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:38.801+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:04:38.813+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:04:38.811+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:38.843+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:04:38.835+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:04:38.848+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:04:38.942+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.200 seconds
[2024-06-13T21:05:09.099+0000] {processor.py:161} INFO - Started process (PID=870) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:09.103+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:05:09.109+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:05:09.108+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:09.138+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:05:09.132+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:05:09.142+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:09.204+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.126 seconds
[2024-06-13T21:05:39.455+0000] {processor.py:161} INFO - Started process (PID=881) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:39.459+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:05:39.465+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:05:39.464+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:39.490+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:05:39.484+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:05:39.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:05:39.551+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.114 seconds
[2024-06-13T21:20:48.172+0000] {processor.py:161} INFO - Started process (PID=892) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:20:48.175+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:20:48.182+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:20:48.180+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:20:48.205+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:20:48.193+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:20:48.207+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:20:48.311+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.156 seconds
[2024-06-13T21:21:18.736+0000] {processor.py:161} INFO - Started process (PID=904) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:21:18.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:21:18.750+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:21:18.749+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:21:18.782+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:21:18.774+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:21:18.786+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:21:18.867+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.157 seconds
[2024-06-13T21:25:10.051+0000] {processor.py:161} INFO - Started process (PID=911) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:10.055+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:25:10.058+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:25:10.058+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:10.096+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:25:10.075+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:25:10.098+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:10.150+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.121 seconds
[2024-06-13T21:25:40.691+0000] {processor.py:161} INFO - Started process (PID=927) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:40.694+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:25:40.700+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:25:40.699+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:40.725+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:25:40.719+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:25:40.729+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:25:40.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.122 seconds
[2024-06-13T21:31:20.045+0000] {processor.py:161} INFO - Started process (PID=932) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:20.048+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:31:20.058+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:31:20.057+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:20.084+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:31:20.080+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:31:20.088+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:20.587+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.568 seconds
[2024-06-13T21:31:51.226+0000] {processor.py:161} INFO - Started process (PID=950) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:51.230+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:31:51.237+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:31:51.236+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:51.267+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:31:51.259+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:31:51.270+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:31:51.369+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.165 seconds
[2024-06-13T21:35:43.173+0000] {processor.py:161} INFO - Started process (PID=956) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:35:43.175+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:35:43.206+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:35:43.204+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:35:43.239+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:35:43.234+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:35:43.240+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:35:43.337+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.272 seconds
[2024-06-13T21:36:13.604+0000] {processor.py:161} INFO - Started process (PID=973) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:13.613+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:36:13.625+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:36:13.623+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:13.668+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:36:13.660+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:36:13.673+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:13.799+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.341 seconds
[2024-06-13T21:36:44.463+0000] {processor.py:161} INFO - Started process (PID=984) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:44.467+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:36:44.475+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:36:44.474+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:44.514+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:36:44.506+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:36:44.519+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:36:44.613+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.186 seconds
[2024-06-13T21:37:14.823+0000] {processor.py:161} INFO - Started process (PID=995) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:14.826+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:37:14.833+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:37:14.832+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:14.855+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:37:14.848+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:37:14.860+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:14.944+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.149 seconds
[2024-06-13T21:37:45.204+0000] {processor.py:161} INFO - Started process (PID=1006) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:45.209+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:37:45.219+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:37:45.217+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:45.248+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:37:45.240+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:37:45.253+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:37:45.343+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.176 seconds
[2024-06-13T21:38:15.775+0000] {processor.py:161} INFO - Started process (PID=1018) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:15.781+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:38:15.788+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:38:15.787+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:15.816+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:38:15.810+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:38:15.820+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:15.891+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.140 seconds
[2024-06-13T21:38:46.177+0000] {processor.py:161} INFO - Started process (PID=1029) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:46.180+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:38:46.189+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:38:46.188+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:46.215+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:38:46.210+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:38:46.219+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:38:46.288+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.132 seconds
[2024-06-13T21:39:16.628+0000] {processor.py:161} INFO - Started process (PID=1040) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:39:16.632+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:39:16.640+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:39:16.639+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:39:16.663+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:39:16.658+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:39:16.666+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:39:16.729+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.127 seconds
[2024-06-13T21:43:14.784+0000] {processor.py:161} INFO - Started process (PID=1046) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:14.793+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:43:14.803+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:43:14.802+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:14.826+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:43:14.823+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:43:14.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:14.980+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.337 seconds
[2024-06-13T21:43:45.464+0000] {processor.py:161} INFO - Started process (PID=1064) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:45.470+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:43:45.479+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:43:45.477+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:45.510+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:43:45.503+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:43:45.515+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:43:45.603+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.173 seconds
[2024-06-13T21:46:07.107+0000] {processor.py:161} INFO - Started process (PID=1069) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:07.109+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:46:07.113+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:46:07.112+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:07.127+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:46:07.124+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:46:07.129+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:07.167+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.222 seconds
[2024-06-13T21:46:37.573+0000] {processor.py:161} INFO - Started process (PID=1087) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:37.577+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:46:37.584+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:46:37.583+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:37.613+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:46:37.607+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:46:37.618+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:46:37.690+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.140 seconds
[2024-06-13T21:50:55.280+0000] {processor.py:161} INFO - Started process (PID=1103) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:50:55.282+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:50:55.285+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:50:55.284+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:50:55.297+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:50:55.293+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:50:55.298+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:50:55.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.060 seconds
[2024-06-13T21:51:25.922+0000] {processor.py:161} INFO - Started process (PID=1114) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:25.926+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:51:25.934+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:51:25.933+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:25.960+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:51:25.954+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:51:25.964+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:26.045+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.147 seconds
[2024-06-13T21:51:56.360+0000] {processor.py:161} INFO - Started process (PID=1125) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:56.363+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:51:56.371+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:51:56.370+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:56.400+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:51:56.392+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:51:56.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:51:56.488+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.158 seconds
[2024-06-13T21:52:27.430+0000] {processor.py:161} INFO - Started process (PID=1136) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:27.434+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:52:27.441+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:52:27.440+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:27.465+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:52:27.459+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:52:27.469+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:27.534+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.126 seconds
[2024-06-13T21:52:57.888+0000] {processor.py:161} INFO - Started process (PID=1147) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:57.896+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:52:57.915+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:52:57.913+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:57.957+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:52:57.946+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:52:57.961+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:52:58.069+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.240 seconds
[2024-06-13T21:53:28.314+0000] {processor.py:161} INFO - Started process (PID=1158) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:28.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:53:28.332+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:53:28.331+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:28.359+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:53:28.352+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:53:28.362+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:28.449+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.169 seconds
[2024-06-13T21:53:58.862+0000] {processor.py:161} INFO - Started process (PID=1169) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:58.867+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:53:58.874+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:53:58.873+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:58.897+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:53:58.892+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:53:58.900+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:53:58.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.121 seconds
[2024-06-13T21:54:29.185+0000] {processor.py:161} INFO - Started process (PID=1180) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:29.188+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:54:29.195+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:54:29.194+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:29.210+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:54:29.206+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:54:29.212+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:29.269+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.103 seconds
[2024-06-13T21:54:59.594+0000] {processor.py:161} INFO - Started process (PID=1191) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:59.596+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:54:59.601+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:54:59.600+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:59.614+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:54:59.611+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:54:59.617+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:54:59.668+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.091 seconds
[2024-06-13T21:55:29.939+0000] {processor.py:161} INFO - Started process (PID=1202) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:55:29.943+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T21:55:29.950+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:55:29.949+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:55:29.971+0000] {logging_mixin.py:188} INFO - [2024-06-13T21:55:29.966+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T21:55:29.974+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T21:55:30.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.112 seconds
[2024-06-13T22:13:05.878+0000] {processor.py:161} INFO - Started process (PID=1215) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:05.881+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:13:05.888+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:13:05.886+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:05.941+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:13:05.932+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:13:05.944+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:06.014+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.192 seconds
[2024-06-13T22:13:36.379+0000] {processor.py:161} INFO - Started process (PID=1226) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:36.383+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:13:36.394+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:13:36.392+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:36.431+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:13:36.421+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:13:36.437+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:13:36.552+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.217 seconds
[2024-06-13T22:14:07.024+0000] {processor.py:161} INFO - Started process (PID=1237) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:07.027+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:14:07.034+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:14:07.033+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:07.058+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:14:07.052+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:14:07.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:07.132+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.131 seconds
[2024-06-13T22:14:37.413+0000] {processor.py:161} INFO - Started process (PID=1248) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:37.418+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:14:37.463+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:14:37.461+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:37.493+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:14:37.487+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:14:37.498+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:14:37.561+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.172 seconds
[2024-06-13T22:15:07.841+0000] {processor.py:161} INFO - Started process (PID=1259) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:07.843+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:15:07.849+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:15:07.848+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:07.867+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:15:07.862+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:15:07.871+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:07.935+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.116 seconds
[2024-06-13T22:15:38.222+0000] {processor.py:161} INFO - Started process (PID=1270) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:38.225+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:15:38.231+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:15:38.230+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:38.249+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:15:38.245+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:15:38.252+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:15:38.310+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.112 seconds
[2024-06-13T22:16:08.576+0000] {processor.py:161} INFO - Started process (PID=1281) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:16:08.579+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:16:08.586+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:16:08.584+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:16:08.614+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:16:08.608+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:16:08.618+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:16:08.684+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.131 seconds
[2024-06-13T22:32:33.686+0000] {processor.py:161} INFO - Started process (PID=1293) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:32:33.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:32:33.720+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:32:33.718+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:32:33.771+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:32:33.760+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:32:33.773+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:32:33.897+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.282 seconds
[2024-06-13T22:33:04.586+0000] {processor.py:161} INFO - Started process (PID=1304) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:04.591+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:33:04.603+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:33:04.601+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:04.643+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:33:04.633+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:33:04.648+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:04.743+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.189 seconds
[2024-06-13T22:33:35.037+0000] {processor.py:161} INFO - Started process (PID=1315) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:35.041+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:33:35.052+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:33:35.050+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:35.082+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:33:35.076+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:33:35.087+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:33:35.168+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.160 seconds
[2024-06-13T22:34:05.637+0000] {processor.py:161} INFO - Started process (PID=1326) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:05.642+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:34:05.651+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:34:05.650+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:05.680+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:34:05.674+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:34:05.684+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:05.752+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.148 seconds
[2024-06-13T22:34:36.036+0000] {processor.py:161} INFO - Started process (PID=1337) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:36.039+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:34:36.046+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:34:36.045+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:36.066+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:34:36.060+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:34:36.069+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:34:36.145+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.133 seconds
[2024-06-13T22:35:06.509+0000] {processor.py:161} INFO - Started process (PID=1348) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:06.511+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:35:06.519+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:35:06.518+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:06.543+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:35:06.537+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:35:06.547+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:06.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.134 seconds
[2024-06-13T22:35:36.853+0000] {processor.py:161} INFO - Started process (PID=1359) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:36.856+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:35:36.862+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:35:36.861+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:36.881+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:35:36.876+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:35:36.884+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:35:36.939+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.105 seconds
[2024-06-13T22:36:07.253+0000] {processor.py:161} INFO - Started process (PID=1370) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:07.255+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:36:07.261+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:36:07.260+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:07.279+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:36:07.275+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:36:07.282+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:07.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.099 seconds
[2024-06-13T22:36:37.594+0000] {processor.py:161} INFO - Started process (PID=1382) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:37.596+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:36:37.601+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:36:37.600+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:37.616+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:36:37.612+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:36:37.618+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:36:37.669+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.094 seconds
[2024-06-13T22:51:40.854+0000] {processor.py:161} INFO - Started process (PID=1393) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:51:40.868+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:51:40.875+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:51:40.874+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:51:40.958+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:51:40.939+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:51:40.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:51:40.996+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.155 seconds
[2024-06-13T22:52:11.445+0000] {processor.py:161} INFO - Started process (PID=1404) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:52:11.449+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T22:52:11.458+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:52:11.456+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:52:11.486+0000] {logging_mixin.py:188} INFO - [2024-06-13T22:52:11.478+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T22:52:11.492+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T22:52:11.570+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.152 seconds
[2024-06-13T23:07:29.310+0000] {processor.py:161} INFO - Started process (PID=1411) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:07:29.312+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:07:29.325+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:07:29.324+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:07:29.350+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:07:29.347+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:07:29.352+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:07:29.825+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.530 seconds
[2024-06-13T23:08:00.619+0000] {processor.py:161} INFO - Started process (PID=1427) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:08:00.623+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:08:00.633+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:08:00.631+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:08:00.655+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:08:00.649+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:08:00.661+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:08:00.734+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.142 seconds
[2024-06-13T23:23:20.492+0000] {processor.py:161} INFO - Started process (PID=1445) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:20.494+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:23:20.498+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:23:20.497+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:20.509+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:23:20.506+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:23:20.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:20.547+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.066 seconds
[2024-06-13T23:23:51.190+0000] {processor.py:161} INFO - Started process (PID=1456) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:51.195+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:23:51.203+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:23:51.201+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:51.229+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:23:51.222+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:23:51.234+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:23:51.302+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.137 seconds
[2024-06-13T23:40:05.586+0000] {processor.py:161} INFO - Started process (PID=1468) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:05.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:40:05.615+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:40:05.614+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:05.688+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:40:05.678+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:40:05.691+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:05.768+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.325 seconds
[2024-06-13T23:40:36.279+0000] {processor.py:161} INFO - Started process (PID=1479) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:36.282+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:40:36.289+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:40:36.288+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:36.321+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:40:36.310+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:40:36.325+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:40:36.408+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.156 seconds
[2024-06-13T23:41:06.892+0000] {processor.py:161} INFO - Started process (PID=1490) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:06.895+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:41:06.904+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:41:06.903+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:06.930+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:41:06.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:41:06.934+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:07.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.165 seconds
[2024-06-13T23:41:37.280+0000] {processor.py:161} INFO - Started process (PID=1501) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:37.284+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:41:37.292+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:41:37.291+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:37.315+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:41:37.309+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:41:37.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:41:37.389+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.131 seconds
[2024-06-13T23:42:07.756+0000] {processor.py:161} INFO - Started process (PID=1512) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:07.759+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:42:07.766+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:42:07.765+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:07.788+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:42:07.782+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:42:07.791+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:07.856+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.126 seconds
[2024-06-13T23:42:38.102+0000] {processor.py:161} INFO - Started process (PID=1523) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:38.105+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:42:38.110+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:42:38.109+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:38.127+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:42:38.122+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:42:38.129+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:42:38.203+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.119 seconds
[2024-06-13T23:43:08.509+0000] {processor.py:161} INFO - Started process (PID=1534) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:08.512+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:43:08.518+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:43:08.517+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:08.536+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:43:08.532+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:43:08.539+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:08.595+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.102 seconds
[2024-06-13T23:43:38.835+0000] {processor.py:161} INFO - Started process (PID=1545) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:38.838+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:43:38.843+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:43:38.842+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:38.861+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:43:38.857+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:43:38.863+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:43:38.916+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.104 seconds
[2024-06-13T23:45:34.432+0000] {processor.py:161} INFO - Started process (PID=1556) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:45:34.434+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:45:34.438+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:45:34.437+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:45:34.451+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:45:34.447+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:45:34.453+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:45:34.500+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.086 seconds
[2024-06-13T23:46:05.023+0000] {processor.py:161} INFO - Started process (PID=1567) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:46:05.026+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:46:05.032+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:46:05.031+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:46:05.049+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:46:05.044+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:46:05.051+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:46:05.111+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.109 seconds
[2024-06-13T23:52:42.923+0000] {processor.py:161} INFO - Started process (PID=1583) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:52:42.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:52:42.930+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:52:42.930+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:52:42.939+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:52:42.936+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:52:42.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:52:42.983+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.079 seconds
[2024-06-13T23:53:13.832+0000] {processor.py:161} INFO - Started process (PID=1594) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:13.838+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:53:13.847+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:53:13.845+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:13.875+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:53:13.868+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:53:13.879+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:13.948+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.152 seconds
[2024-06-13T23:53:44.255+0000] {processor.py:161} INFO - Started process (PID=1605) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:44.258+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:53:44.267+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:53:44.265+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:44.289+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:53:44.284+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:53:44.293+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:53:44.356+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.120 seconds
[2024-06-13T23:54:14.651+0000] {processor.py:161} INFO - Started process (PID=1615) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:14.655+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:54:14.662+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:54:14.660+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:14.684+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:54:14.678+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:54:14.687+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:14.750+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.118 seconds
[2024-06-13T23:54:45.048+0000] {processor.py:161} INFO - Started process (PID=1626) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:45.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:54:45.058+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:54:45.056+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:45.085+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:54:45.079+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:54:45.089+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:54:45.157+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.130 seconds
[2024-06-13T23:55:15.476+0000] {processor.py:161} INFO - Started process (PID=1637) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:15.480+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:55:15.486+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:55:15.485+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:15.508+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:55:15.503+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:55:15.511+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:15.598+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.144 seconds
[2024-06-13T23:55:45.880+0000] {processor.py:161} INFO - Started process (PID=1647) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:45.884+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:55:45.891+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:55:45.890+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:45.912+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:55:45.907+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:55:45.915+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:55:45.984+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.128 seconds
[2024-06-13T23:56:16.285+0000] {processor.py:161} INFO - Started process (PID=1658) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:16.289+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:56:16.295+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:56:16.294+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:16.315+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:56:16.310+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:56:16.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:16.383+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.122 seconds
[2024-06-13T23:56:46.709+0000] {processor.py:161} INFO - Started process (PID=1669) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:46.713+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:56:46.722+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:56:46.721+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:46.753+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:56:46.748+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:56:46.757+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:56:46.820+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.128 seconds
[2024-06-13T23:57:17.088+0000] {processor.py:161} INFO - Started process (PID=1680) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:17.091+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:57:17.097+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:57:17.096+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:17.116+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:57:17.111+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:57:17.120+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:17.179+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.111 seconds
[2024-06-13T23:57:47.599+0000] {processor.py:161} INFO - Started process (PID=1691) to work on /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:47.602+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/spotify_dag.py for tasks to queue
[2024-06-13T23:57:47.612+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:57:47.611+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:47.646+0000] {logging_mixin.py:188} INFO - [2024-06-13T23:57:47.635+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/spotify_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/spotify_dag.py", line 4, in <module>
    from spotify_etl import run_spotify_etl
ModuleNotFoundError: No module named 'spotify_etl'
[2024-06-13T23:57:47.652+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/spotify_dag.py
[2024-06-13T23:57:47.842+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/spotify_dag.py took 0.283 seconds
